### ISSUE TITLE
KafkaConsumer: InterruptedException not handled properly

### ISSUE THREAD

**Issue by RolefH** [IBMStreams/streamsx.kafka#75]
Following Exception should not be reported as an error:

```
 ERROR #splapptrc,J[11],P[67],MessagehubStream M[AbstractKafkaConsumerOperator.java:com.ibm.streamsx.kafka.operators.AbstractKafkaConsumerOperator$1.run:461]  - Operator error
 ERROR #splapptrc,J[11],P[67],MessagehubStream M[?:?:0]  - java.lang.RuntimeException: Consumer interrupted while waiting for messages to arrive
 ERROR #splapptrc,J[11],P[67],MessagehubStream M[?:?:0]  -  at com.ibm.streamsx.kafka.clients.consumer.KafkaConsumerClient.getNextRecord(KafkaConsumerClient.java:546)
 ERROR #splapptrc,J[11],P[67],MessagehubStream M[?:?:0]  -  at com.ibm.streamsx.kafka.operators.AbstractKafkaConsumerOperator.produceTuples(AbstractKafkaConsumerOperator.java:512)
 ERROR #splapptrc,J[11],P[67],MessagehubStream M[?:?:0]  -  at com.ibm.streamsx.kafka.operators.AbstractKafkaConsumerOperator.access$000(AbstractKafkaConsumerOperator.java:47)
 ERROR #splapptrc,J[11],P[67],MessagehubStream M[?:?:0]  -  at com.ibm.streamsx.kafka.operators.AbstractKafkaConsumerOperator$1.run(AbstractKafkaConsumerOperator.java:459)
 ERROR #splapptrc,J[11],P[67],MessagehubStream M[?:?:0]  -  at java.lang.Thread.run(Thread.java:785)
 ERROR #splapptrc,J[11],P[67],MessagehubStream M[?:?:0]  -  at com.ibm.streams.operator.internal.runtime.OperatorThreadFactory$2.run(OperatorThreadFactory.java:137)
 ERROR #splapptrc,J[11],P[67],MessagehubStream M[?:?:0]  - Caused by: java.lang.InterruptedException
 ERROR #splapptrc,J[11],P[67],MessagehubStream M[?:?:0]  -  at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.reportInterruptAfterWait(AbstractQueuedSynchronizer.java:2025)
 ERROR #splapptrc,J[11],P[67],MessagehubStream M[?:?:0]  -  at java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:2099)
 ERROR #splapptrc,J[11],P[67],MessagehubStream M[?:?:0]  -  at java.util.concurrent.LinkedBlockingQueue.poll(LinkedBlockingQueue.java:478)
 ERROR #splapptrc,J[11],P[67],MessagehubStream M[?:?:0]  -  at com.ibm.streamsx.kafka.clients.consumer.KafkaConsumerClient.getNextRecord(KafkaConsumerClient.java:544)
 ERROR #splapptrc,J[11],P[67],MessagehubStream M[?:?:0]  -  ... 5 more
```
The thread listening at the internal queue gets interrupted on shutdown. This is normal shutdown behaviour. The InterruptedException is caught, but re-thrown as a RuntimeException. This exception handling is wrong.

**Comment by RolefH** [comment]
Seen in MessageHubConsumer, which is based on Kafka Toolkit.

**Comment by RolefH** [comment]
fixed in release v1.3.1
